src/components/tasks/SuggestionBox/SuggestionList.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import classNames from '@/components/tasks/SuggestionBox/suggestion.module.scss';
+
```

Comment:
please  use another name instead of classNames as it is an HTML attribute in React

Reply:
This is the name used across the whole repo for import classes from modules. @Pratiyushkumar